pkg/dbutil/query.go

```diff
@@ -0,0 +1,66 @@
+package dbutil
+
+import (
+	"database/sql"
+
+	"github.com/juju/errors"
+	"github.com/ngaut/log"
+)
+
+// QuerySQL queries sql, and returns some row
+func QuerySQL(db *sql.DB, query string) (*sql.Rows, error) {
+	log.Debugf("[query][sql] %s", query)
+	rows, err := db.Query(query)
+	if err != nil {
+		log.Errorf("query sql[%s] failed %v", query, errors.ErrorStack(err))
+		return nil, errors.Trace(err)
+	}
```

Comment:
maybe we can just return the error but do not log it and let outer caller to log the error?

Reply:
yep, I like it, let's define a log specification